### PR TITLE
Add Mastodon/Pleroma support

### PR DESCRIPTION
**Pull request by vladimyr**
Fetch followers for a given user from the default (mastodon.social) or custom Mastodon/Pleroma instance.

This adds a new handler:
```
/mastodon/follow/:userId/:instance?
```

## Preview

image

As proposed here: https://github.com/badgen/badgen.net/issues/362#issuecomment-615880134

### CODE REVIEW

File: api/mastodon.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    '/mastodon/follow/:userId/:instance?': handler
+  }
+})
+
+async function handler ({ userId, instance = 'mastodon.social' }: PathArgs) {
+  const prefixUrl = `https://${instance}/api/v1/`
+  const info = await got(`accounts/${userId}`, { prefixUrl }).json<any>()
+
+  return {
+    subject: `follow @${info.username}@${instance}`,
```
Comment (amio) [inline api/mastodon.ts:22]: Two `@` looks a bit duplicate, maybe remove the first one?
```suggestion
    subject: `follow ${info.username}@${instance}`,
```

File: api/mastodon.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import got from '../libs/got'
+import { millify } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+export default createBadgenHandler({
+  title: 'Mastodon/Pleroma',
+  examples: {
+    '/mastodon/follow/1': 'followers count (mastodon.social)',
+    '/mastodon/follow/1/mas.to': 'followers count (custom Mastodon instance)',
+    '/mastodon/follow/1/pleroma.site': 'followers count (custom Pleroma instance)',
```
Comment (amio) [inline api/mastodon.ts:10]: It would be nice to make the description shorter, avoid that overflow in screenshot :)

File: api/mastodon.ts (hunk 1 above)
Comment (vladimyr) [reply api/mastodon.ts:22]: I agree but that is a convention in Mastodon world:



https://mastodon.social/@Gargron

File: api/mastodon.ts (hunk 2 above)
Comment (vladimyr) [reply api/mastodon.ts:10]: image

Better? 👨‍🎨

Comment (amio) [reply api/mastodon.ts]: Looks nice!

File: api/mastodon.ts (hunk 1 above)
Comment (amio) [reply api/mastodon.ts:22]: Yea, let's go with convention 👍

**Comment by amio** [comment]
Thanks! @vladimyr

**Pull request merged**